# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpBackgroundChannelChild.h
# Commit: d59b42f58f86
# Full Hash: d59b42f58f861e049c1270035ea7568acc0605b9
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P3 send OS*R and ODA via pHttpChannel for multipart channel, r=mayhemer,necko-reviewers
#   
#   Depends on D76969
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D76970
# ==============================================================================

diff -r cbcd3ee87d28 -r d59b42f58f86 netwerk/protocol/http/HttpBackgroundChannelChild.h
--- a/netwerk/protocol/http/HttpBackgroundChannelChild.h	Tue Jun 30 00:32:38 2020 +0000
+++ b/netwerk/protocol/http/HttpBackgroundChannelChild.h	Tue Jun 30 00:32:53 2020 +0000
@@ -44,7 +44,7 @@
 
   // Callback when OnStartRequest is received and handled by HttpChannelChild.
   // Enqueued messages in background channel will be flushed.
-  void OnStartRequestReceived();
+  void OnStartRequestReceived(Maybe<uint32_t> aMultiPartID);
 
 #ifdef MOZ_DIAGNOSTIC_ASSERT_ENABLED
   bool IsQueueEmpty() const { return mQueuedRunnables.IsEmpty(); }